src/marketing/gift-registries.md

```diff
@@ -5,8 +5,13 @@ title: Gift Registries
 
 Magento gives your customers the ability to create gift registries for special occasions, and to invite their friends and family to purchase their gifts from the gift registry. Magento keeps track of all items purchased and the quantities remaining.
 
-![Example storefront - gift registry]({% link images/images-ee/storefront-gift-registry-create-baby-info.png %}){: .zoom}
-_Create Gift Registry in Customer Dashboard_
+<!--{% if "Default.EE Only" contains site.edition %}-->
+![]({% link images/images-ee/storefront-gift-registry-create-baby-info.png %}){: .zoom}
+<!--{% endif %}-->
+<!--{% if "Default.B2B Only" contains site.edition %}-->
+![]({% link images/images-b2b/storefront-gift-registry-create-baby-info.png %}){: .zoom}
+<!--{% endif %}-->
+_Creating a Baby Gift Registry_
 
```

Comment:
We don't want to get rid of the alt text here. We are in the process of putting them in. The `[Example storefront - gift registry]` is basically meta data that affects our SEO. It does not necessarily match the caption. Going forward we want all images to have alt text.